source/features/center-reactions-popup.css

```diff
@@ -55,5 +54,5 @@ td.js-line-comments:last-child .timeline-comment-actions .add-reaction-popover::
 /* Keep colored button when reaction popup is open */
 .dropdown-details[open] .timeline-comment-action {
 	opacity: 100%;
-	color: var(--color-mktg-btn-bg, #4078c0); /* GHE #4121 */
```

Comment:
I also dropped every such comment. The idea was that fallback colors were just for GHE, but in reality GH keeps changing the class names…

cc @kidonng the class checker might not be too difficult to create:

1. Find `var([^,]+` in our repo
2. Fetch GitHub’s CSS file (if static)
3. Look for each variable